openlibrary/app.py

```diff
@@ -66,3 +71,42 @@ def render_template(name, *a, **kw):
     if "." in name:
         name = name.rsplit(".", 1)[0]
     return render[name](*a, **kw)
+
+
+mc_cache = MemcacheCache()
+
+
+@macro
+@public
+def render_macro_lazy(name, *args, **kw):
```

Comment:
Are these two functions the same without the exception of one string?  If so, would it be possible to add a parameter or use [functools.partial](https://docs.python.org/3/library/functools.html#functools.partial) to merge this into a single function?